assets/src/edit-story/masks/frame.js

```diff
@@ -84,17 +92,35 @@ function WithDropTarget({ element, children }) {
         height="100%"
         preserveAspectRatio="none"
       >
+        {/** Suble indicator that the element has a drop target */}
+        <DropTargetPath
+          ref={indicatorRef}
+          vectorEffect="non-scaling-stroke"
+          strokeWidth="4"
+          fill="none"
+          stroke="#0063F9"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          d={mask?.path}
+          style={
+            (hover && !draggingResource) ||
+            (Boolean(draggingResource) &&
+              isDropSource(draggingResource.type) &&
+              draggingResource !== resource)
+              ? { opacity: 1 }
+              : {}
```

Comment:
So, the default `opacity: 0` right? Where is it set?

Reply:
It's already set in the styled component `<DropTargetPath />`